{
  "cveMetadata": {
    "cveId": "CVE-2023-6507",
    "assignerOrgId": "28c92f92-d60d-412d-b760-e73465c3df22",
    "state": "PUBLISHED",
    "assignerShortName": "PSF",
    "dateReserved": "2023-12-04T21:24:50.284Z",
    "datePublished": "2023-12-08T18:20:49.583Z",
    "dateUpdated": "2024-08-02T08:35:13.267Z"
  },
  "containers": {
    "adp": [
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2023-6507",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "total"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-01-31T20:16:20.207522Z"
              }
            }
          }
        ],
        "affected": [
          {
            "cpes": [
              "cpe:2.3:a:python:cpython:3.13.0:*:*:*:*:*:*:*"
            ],
            "vendor": "python",
            "product": "cpython",
            "versions": [
              {
                "status": "affected",
                "version": "3.13.0"
              }
            ],
            "defaultStatus": "unknown"
          },
          {
            "cpes": [
              "cpe:2.3:a:python:cpython:*:*:*:*:*:*:*:*"
            ],
            "vendor": "python",
            "product": "cpython",
            "versions": [
              {
                "status": "affected",
                "version": "3.12.0"
              }
            ],
            "defaultStatus": "unknown"
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-06-04T17:17:02.629Z"
        }
      },
      {
        "providerMetadata": {
          "orgId": "af854a3a-2127-422b-91ae-364da2661108",
          "shortName": "CVE",
          "dateUpdated": "2024-08-02T08:35:13.267Z"
        },
        "title": "CVE Program Container",
        "references": [
          {
            "tags": [
              "issue-tracking",
              "x_transferred"
            ],
            "url": "https://github.com/python/cpython/issues/112334"
          },
          {
            "tags": [
              "vendor-advisory",
              "x_transferred"
            ],
            "url": "https://mail.python.org/archives/list/security-announce@python.org/thread/AUL7QFHBLILGISS7U63B47AYSSGJJQZD/"
          },
          {
            "tags": [
              "patch",
              "x_transferred"
            ],
            "url": "https://github.com/python/cpython/commit/9fe7655c6ce0b8e9adc229daf681b6d30e6b1610"
          },
          {
            "tags": [
              "patch",
              "x_transferred"
            ],
            "url": "https://github.com/python/cpython/commit/10e9bb13b8dcaa414645b9bd10718d8f7179e82b"
          },
          {
            "tags": [
              "patch",
              "x_transferred"
            ],
            "url": "https://github.com/python/cpython/commit/85bbfa8a4bbdbb61a3a84fbd7cb29a4096ab8a06"
          }
        ]
      }
    ]
  },
  "search": "解析验收测试文案"
}